test/libyul/ewasmTranslationTests/memoryinit.yul

```diff
@@ -0,0 +1,10 @@
+{
+  memoryinit(0x0102030405060708090a0b0c0d0e0f101112131415161718191a1b1c1d1e1f20)
+  sstore(1, mload(0x40))
+}
+// ----
+// Trace:
+// Memory dump:
+//     40: 0102030405060708090a0b0c0d0e0f101112131415161718191a1b1c1d1e1f20
+// Storage dump:
+//   0000000000000000000000000000000000000000000000000000000000000001: 0102030405060708090a0b0c0d0e0f101112131415161718191a1b1c1d1e1f20
```

Comment:
This test was working earlier - now it's failing in the weirdest way... I'm not quite sure yet, what's the cause of that...

Reply:
Is this rebased on top of my refactoring?

Reply:
Which refactoring exactly? It's rebased to develop... I mean, I'm just seeing that I didn't implement ``memoryinit`` properly, but the ``sstore`` should still not vanish because of it... so not sure yet, what's happening, still looking into it...